Dockerfile

```diff
@@ -1288,17 +1288,11 @@ COPY --chown=airflow:0 ${AIRFLOW_SOURCES_FROM} ${AIRFLOW_SOURCES_TO}
 # Add extra python dependencies
 ARG ADDITIONAL_PYTHON_DEPS=""
 
-# Those are additional constraints that are needed for some extras but we do not want to
```

Comment:
This part is removed from PROD image because EAGER_UPGRADE_ADDITIONAL_REQUIREMENTS is. only used in CI image.